weblate/settings.py

```diff
@@ -342,6 +318,7 @@
     'weblate.trans',
     'weblate.lang',
     'weblate.accounts',
+    'libravatar',
```

Comment:
If it was like this, it's clearly a bug in that page, I'll take a look at it :-).